exporting/prometheus/remote_write/README.md

```diff
@@ -0,0 +1,38 @@
+# Prometheus remote write exporting connector
+
```

Comment:
Could we add a new sentence here that very briefly explains what this connector does?

The Prometheus remote write exporting connector uses the [exporting engine](../README.md) to send Netdata metrics to your choice of more than 20 external storage providers for long-term archiving and further analysis.

Reply:
Done